### PR TITLE
Release bigquery 1.7.0

### PR DESCRIPTION
**Pull request by shollyman**
This pull request was generated using releasetool.

### CODE REVIEW

File: bigquery/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,35 @@
...
+### Documentation
+
+- Fix [#6321](https://github.com/googleapis/google-cloud-python/pull/6321) Update README service links in quickstart guides. ([#6322](https://github.com/googleapis/google-cloud-python/pull/6322))
+- Move usage guides to their own docs. ([#6238](https://github.com/googleapis/google-cloud-python/pull/6238))
+- Normalize use of support level badges ([#6159](https://github.com/googleapis/google-cloud-python/pull/6159))
+
+### Internal / Testing Changes
+
+- Deprecation cleanups ([#6304](https://github.com/googleapis/google-cloud-python/pull/6304))
+- Use _get_sub_prop helper so missing load stats don't raise. ([#6269](https://github.com/googleapis/google-cloud-python/pull/6269))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]: Markdown fix:
```suggestion
- Use `_get_sub_prop` helper so missing load stats don't raise. ([#6269](https://github.com/googleapis/google-cloud-python/pull/6269))
```

File: bigquery/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,35 @@
...
+### Implementation Changes
+
+- Add destination table properties to 'LoadJobConfig'. ([#6202](https://github.com/googleapis/google-cloud-python/pull/6202))
+- Allow strings or references in create_dataset and create_table ([#6199](https://github.com/googleapis/google-cloud-python/pull/6199))
+- Fix swallowed error message ([#6168](https://github.com/googleapis/google-cloud-python/pull/6168))
+
+### New Features
+
+- Add --params option to %%bigquery magic ([#6277](https://github.com/googleapis/google-cloud-python/pull/6277))
+- Expose 'to_api_repr' method for jobs. ([#6176](https://github.com/googleapis/google-cloud-python/pull/6176))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]: Markdown fix:
```suggestion
- Expose `to_api_repr` method for jobs. ([#6176](https://github.com/googleapis/google-cloud-python/pull/6176))
```

File: bigquery/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,35 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery/#history
 
+## 1.7.0
+
+11-05-2018 16:41 PST
+
+### Implementation Changes
+
+- Add destination table properties to 'LoadJobConfig'. ([#6202](https://github.com/googleapis/google-cloud-python/pull/6202))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]: Markdown fix:
```suggestion
- Add destination table properties to `LoadJobConfig`. ([#6202](https://github.com/googleapis/google-cloud-python/pull/6202))
```

File: bigquery/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,35 @@
...
+- Allow strings or references in create_dataset and create_table ([#6199](https://github.com/googleapis/google-cloud-python/pull/6199))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]: Markdown fix:
```suggestion
- Allow strings or references in `create_dataset` and `create_table` ([#6199](https://github.com/googleapis/google-cloud-python/pull/6199))
```

File: bigquery/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,35 @@
...
+- Fix swallowed error message ([#6168](https://github.com/googleapis/google-cloud-python/pull/6168))
+
+### New Features
+
+- Add --params option to %%bigquery magic ([#6277](https://github.com/googleapis/google-cloud-python/pull/6277))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]: Markdown fix:
```suggestion
- Add `--params` option to `%%bigquery` magic ([#6277](https://github.com/googleapis/google-cloud-python/pull/6277))
```

File: bigquery/CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,35 @@
...
+- Expose 'to_api_repr' method for jobs. ([#6176](https://github.com/googleapis/google-cloud-python/pull/6176))
+- Allow string in addition to DatasetReference / TableReference in Clie… ([#6164](https://github.com/googleapis/google-cloud-python/pull/6164))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]: Replace ellipsis:  `in Client methods`.

File: bigquery/CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,35 @@
...
+
+### Documentation
+
+- Fix [#6321](https://github.com/googleapis/google-cloud-python/pull/6321) Update README service links in quickstart guides. ([#6322](https://github.com/googleapis/google-cloud-python/pull/6322))
```
Comment (tseaver) [inline bigquery/CHANGELOG.md]:
```suggestion
- Update README service links in quickstart guides. ([#6322](https://github.com/googleapis/google-cloud-python/pull/6322))
```

**Comment by shollyman** [comment]
Release is at https://github.com/googleapis/google-cloud-python/releases/tag/bigquery-1.7.0

**Pull request merged**